src/pages/Components/Checkbox/index.md

```diff
@@ -9,37 +9,7 @@ order: 1
 
 <preview path="src/pages/Components/Checkbox/previews/intro" nude="true"></preview>
 
-## Standard checkbox states :
-
-### Hover
-
-When the checkbox is hovered due to user navigation.
-
-<preview path="src/pages/Components/Checkbox/previews/checkbox-hover-state" nude="true"></preview>
-
-### Focus
-
-When the checkbox is in focus. It is generally triggered when the user clicks or taps on an element or selects it with the keyboard's "tab" key.
-
-<preview path="src/pages/Components/Checkbox/previews/checkbox-focus-state" nude="true"></preview>
-
-### Disabled
-When the checkbox field is inactive or inaccessible. It is usually used when the input, for some reason, is forbidden or blocked to be edited.
-
-<preview path="src/pages/Components/Checkbox/previews/checkbox-disabled-state" nude="true"></preview>
-
-### Valid
-When the checkbox is checked
-
-<preview path="src/pages/Components/Checkbox/previews/checkbox-valid-state" nude="true"></preview>
-
-### Invalid
-
-When the checkbox should be checked
-
-<preview path="src/pages/Components/Checkbox/previews/checkbox-invalid-state" nude="true"></preview>
-
-##Do & dont's
+## Do & dont's
```

Comment:
We can add this *Don't* :
"Never use checkboxes if you only need to select one option. Please use radio buttons for this use case."